src/gui/AstroCalcDialog.cpp

```diff
@@ -3555,6 +3558,9 @@ void AstroCalcDialog::generateSolarEclipsesLocal()
 								treeItem->setText(SolarEclipseLocal3rdContact, dash);
 							treeItem->setToolTip(SolarEclipseLocal3rdContact, q_("The time of third contact"));
 							treeItem->setText(SolarEclipseLocalLastContact, QString("%1").arg(localeMgr->getPrintableTimeLocal(JD4)));
+							if (centraleclipse)
```

Comment:
Why not use `if (centraleclipse && JD3>JD4)`?

Reply:
Thanks, I have added else... after that. Decided to remove it and forgot to change it.